src/Microsoft.SqlTools.ServiceLayer/ShowPlan/Contracts/GetExecutionPlanRequest.cs

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System.Collections.Generic;
```

Comment:
these should generally be sorted, though there is also a convention to list System namespaces first, then sort the rest (which this appears to follow).  However the namespaces in the next file are sorted with System being in alphabetical order.  I would recommend to try to be consistent with one approach or the other (at least within a particular PR 😄).